.github/workflows/ci.yml

```diff
@@ -19,14 +19,14 @@ jobs:
         with:
           submodules: recursive
 
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
+
       - uses: ./.github/actions/cleanup
```

Comment:
I think this might nuke your protoc install. I added it here to deal with some out of disk space issues but unclear if we still need it

Reply:
Thanks! Will try after that step